meta-avocado-raspberrypi/conf/machine/avocado-raspberrypi4.conf

```diff
@@ -4,7 +4,8 @@
 
 require conf/machine/include/avocado.inc
 
-MACHINEOVERRIDES =. "raspberrypi4:"
+BOOTVARS = "ubootenv"
+MACHINEOVERRIDES =. "bootvars-${BOOTVARS}:raspberrypi4:"
```

Comment:
Does Peridio depend on U-Boot for OTA updates?

Is that why you need to persist the U-Boot environment on the microSD card?